src/main/java/com/cloudbees/plugins/credentials/CredentialsUseListener.java

```diff
@@ -0,0 +1,29 @@
+package com.cloudbees.plugins.credentials;
+
+import hudson.ExtensionPoint;
+import hudson.model.ModelObject;
+import jenkins.util.Listeners;
+
+
+/**
+ * A Listener to track {@link Credentials } usage.
```

Comment:
```suggestion
 * A Listener to track {@link Credentials} usage.
```

Add `@see` links to `track` methods.